src/sales/order-status.md

```diff
@@ -11,18 +11,18 @@ _Order Status_
 
 ## Predefined Order Status
 
-|Order Status|Status Code|
-|--- |--- |
-|Processing|processing|
-|Suspected Fraud|fraud|
-|Pending Payment|pending_payment|
-|Payment Review|payment_review|
-|Pending|pending|
-|On Hold|holded|
+|Order Status|Status Code||
+|--- |--- |--- |
+|Processing|processing|When the state of new orders is set to “Processing”, the _Automatically Invoice All Items_ option becomes available in the configuration.|
+|Suspected Fraud|fraud|Sometimes orders paid via PayPal, eway or any other payment gateway, mark the orders as Suspected Fraud. This means the order does not have invoice issued and the confirmation email is also not sent.|
```

Comment:
Same as previous PR, eway is removed from 2.4

```suggestion
|Suspected Fraud|fraud|Sometimes orders paid via PayPal or another payment gateway are marked as Suspected Fraud. This means the order does not have invoice issued and the confirmation email is also not sent.|
```

Reply:
Done!